src/lib.rs

```diff
@@ -249,27 +243,17 @@
 //! }
 //! ```
 
-#![no_std]
+#![cfg_attr(not(test), no_std)]
 #![doc(html_root_url = "https://docs.rs/bitflags/1.2.1")]
 
-#[cfg(test)]
-#[macro_use]
-extern crate std;
-
-// Re-export libcore using an alias so that the macros can work without
-// requiring `extern crate core` downstream.
-#[doc(hidden)]
-pub extern crate core as _core;
-
```

Comment:
As far as I can tell, this isn't necessary any more. The macros now use `core::` instead of `$crate::_core::`, which seems to work fine, but I'm not entirely sure if it causes problem with macro hygiene.

Reply:
I just noticed that this breaks if someone defines `mod core {}` when using `bitflags!`, I'll put together a PR to fix that.